src/main/java/jenkins/plugins/slack/ActiveNotifier.java

```diff
@@ -46,22 +36,24 @@ public ActiveNotifier(SlackNotifier notifier, BuildListener listener) {
 
     private SlackService getSlack(AbstractBuild r) {
         AbstractProject<?, ?> project = r.getProject();
-        String projectRoom = Util.fixEmpty(project.getProperty(SlackNotifier.SlackJobProperty.class).getRoom());
-        String teamDomain = Util.fixEmpty(project.getProperty(SlackNotifier.SlackJobProperty.class).getTeamDomain());
-        String token = Util.fixEmpty(project.getProperty(SlackNotifier.SlackJobProperty.class).getToken());
+        String projectRoom = Util.fixEmpty(project.getProperty(SlackJobProperty.class).getRoom());
+        String teamDomain = Util.fixEmpty(project.getProperty(SlackJobProperty.class).getTeamDomain());
+        String token = Util.fixEmpty(project.getProperty(SlackJobProperty.class).getToken());
+
```

Comment:
Any particular reason for extracting SlackJobProperty from its containing class?